test/llilc_checkpass.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+#title           :llilc_checkpass.py
+#description     :
+# 
+# llilc_checkpass compares two test rusuts, no matter summary or verbose,
+# checks if any function successfully jitted by LLILC in base result
+# failed jitting in diff result.
+#
+# If the check passed, the script return 0. In case of 0, it might have
+# jitted more functions in diff result than in base result. The two results
+# are not necessary the same in this case. Counting result is reported.
+# 
+# If the check failed due to newly faily routine, return 1.
+#     
+# If the check failed due to unexpected reason, return negative numbers.
+#
+# usage: llilc_checkpass.py [-h] -b BASE_RESULT_PATH -d DIFF_RESULT_PATH
+#
+# optional arguments:
+#   -h, --help            show this help message and exit
+#   -b BASE_RESULT_PATH, --base-result-path BASE_RESULT_PATH
```

Comment:
@AndyAyersMS I like your suggestion about adding high level description of what is going on. I will add it.